dist/css/bootstrap-reboot.min.css.map

```diff
@@ -1 +1 @@
n\nsvg {\n  overflow: hidden;\n  vertical-align: middle;\n}\n\ntable {\n  border-collapse: collapse;\n}\n\ncaption {\n  padding-top: 0.75rem;\n  padding-bottom: 0.75rem;\n  color: #6c757d;\n  text-align: left;\n  caption-side: bottom;\n}\n\nth {\n  text-align: inherit;\n}\n\nlabel {\n  display: inline-block;\n  margin-bottom: 0.5rem;\n}\n\nbutton {\n  border-radius: 0;\n}\n\nbutton:focus {\n  outline: 1px dotted;\n  outline: 5px auto -webkit-focus-ring-color;\n}\n\ninput,\nbutton,\nselect,\noptgroup,\ntextarea {\n  margin: 0;\n  font-family: inherit;\n  font-size: inherit;\n  line-height: inherit;\n}\n\nbutton,\ninput {\n  overflow: visible;\n}\n\nbutton,\nselect {\n  text-transform: none;\n}\n\nbutton,\n[type=\"button\"],\n[type=\"reset\"],\n[type=\"submit\"] {\n  -webkit-appearance: button;\n}\n\nbutton::-moz-focus-inner,\n[type=\"button\"]::-moz-focus-inner,\n[type=\"reset\"]::-moz-focus-inner,\n[type=\"submit\"]::-moz-focus-inner {\n  padding: 0;\n  border-style: none;\n}\n\ninput[type=\"radio\"],\ninput[type=\"checkbox\"] {\n  box-sizing: border-box;\n  padding: 0;\n}\n\ninput[type=\"date\"],\ninput[type=\"time\"],\ninput[type=\"datetime-local\"],\ninput[type=\"month\"] {\n  -webkit-appearance: listbox;\n}\n\ntextarea {\n  overflow: auto;\n  resize: vertical;\n}\n\nfieldset {\n  min-width: 0;\n  padding: 0;\n  margin: 0;\n  border: 0;\n}\n\nlegend {\n  display: block;\n  width: 100%;\n  max-width: 100%;\n  padding: 0;\n  margin-bottom: .5rem;\n  font-size: 1.5rem;\n  line-height: inherit;\n  color: inherit;\n  white-space: normal;\n}\n\nprogress {\n  vertical-align: baseline;\n}\n\n[type=\"number\"]::-webkit-inner-spin-button,\n[type=\"number\"]::-webkit-outer-spin-button {\n  height: auto;\n}\n\n[type=\"search\"] {\n  outline-offset: -2px;\n  -webkit-appearance: none;\n}\n\n[type=\"search\"]::-webkit-search-decoration {\n  -webkit-appearance: none;\n}\n\n::-webkit-file-upload-button {\n  font: inherit;\n  -webkit-appearance: button;\n}\n\noutput {\n  display: inline-block;\n}\n\nsummary {\n  display: list-item;\n  cursor: pointer;\n}\n\ntemplate {\n  display: none;\n}\n\n[hidden] {\n  display: none !important;\n}\n\n/*# sourceMappingURL=bootstrap-reboot.css.map */","// Hover mixin and `$enable-hover-media-query` are deprecated.\n//\n// Originally added during our alphas and maintained during betas, this mixin was\n// designed to prevent `:hover` stickiness on iOS-an issue where hover styles\n// would persist after initial touch.\n//\n// For backward compatibility, we've kept these mixins and updated them to\n// always return their regular pseudo-classes instead of a shimmed media query.\n//\n// Issue: https://github.com/twbs/bootstrap/issues/25195\n\n@mixin hover {\n  &:hover { @content; }\n}\n\n@mixin hover-focus {\n  &:hover,\n  &:focus {\n    @content;\n  }\n}\n\n@mixin plain-hover-focus {\n  &,\n  &:hover,\n  &:focus {\n    @content;\n  }\n}\n\n@mixin hover-focus-active {\n  &:hover,\n  &:focus,\n  &:active {\n    @content;\n  }\n}\n"]}
```

Comment:
@lahmatiy: the only change here I see is the main scss file missing, `bootstrap-reboot.scss` here. You can see the same for docs.min.css.map

Reply:
I see the problem. Will try to solve it!

Reply:
I tried your branch, looks like no problem anymore – just re-run `npm run css-minify`

Reply:
There's definitely a problem. The command to try is `npm run css`. See how the *.min.css.map files don't reference the main scss files nor they have their sources.

Reply:
I get for `bootstrap-grid.min.css.map`
```
{"version":3,"sources":["../../scss/bootstrap-grid.scss","dist/css/bootstrap-grid.css","../../scss/_grid.scss","../../scss/mixins/_grid.scss","../../scss/mixins/_breakpoints.scss","../../scss/mixins/_grid-framework.scss","../../scss/utilities/_display.scss","../../scss/utilities/_flex.scss","../../scss/utilities/_spacing.scss"],"names":[],"mappings":";;;;;;AAOA,I,CACE,qB...
```
It differ from current dist version.
Did you try with CSSO 2.0.1?
Do you run `npm run css` on MacOS or Windows? I suppose it can be the reason if you run on Windows.

Reply:
This is with csso-cli 2.0.1. And yeah, I'm on Windows.

Reply:
@lahmatiy I tested it on Linux and it's fine there, so there's definitely something wrong on Windows.

Reply:
I found the reason and fixed it (https://github.com/css/csso-cli/commit/9523ad6d2647ddff32eb81728b0de495c267ee36). However, a new test I added is failing on latest node.js (both on unix and win) and I don't know why. Will try to fix it and publish a new version then.

Reply:
Fix was released as [2.0.2](https://github.com/css/csso-cli/releases/tag/v2.0.2). Please try it 🙏

Reply:
It seems it's better :) I still see differences in the source maps. I pushed the dist here.

Reply:
The only issue left is the main scss file is missing. For example, for docs, I don't see the docs.sccs file in the sourcemap at all.

I'm still on Windows, but I can try on Linux later again.

Reply:
@lahmatiy: friendly ping :) We only have one issue left as it seems.

Reply:
Sorry for a long response. I'm on vacation and have no access to my laptop. Will come back in 2 days or so, sorry for inconvenience

Reply:
NP, @lahmatiy thanks for the reply :)

Reply:
@lahmatiy ping